docs/remote_inference_blueprints/cohere_connector_embedding_blueprint.md

```diff
@@ -24,7 +24,9 @@ POST /_plugins/_ml/connectors/_create
       "headers": {
         "Authorization": "Bearer ${credential.cohere_key}"
       },
-      "request_body": "{ \"texts\": ${parameters.texts}, \"truncate\": \"${parameters.truncate}\", \"model\": \"${parameters.model}\" }"
+      "request_body": "{ \"texts\": ${parameters.prompt}, \"truncate\": \"${parameters.truncate}\", \"model\": \"${parameters.model}\" }", 
```

Comment:
Cohere expects `texts`, please check [this doc](https://docs.cohere.com/reference/embed)

Reply:
Actually this is a java code issue https://github.com/opensearch-project/ml-commons/blob/2.x/common/src/main/java/org/opensearch/ml/common/connector/MLPreProcessFunction.java#L31

We should fix the java code in next release, but for now let's fix the doc first. When java code fixed, we should change this back to `texts`

Reply:
But that's only if we use `pre_process_function`, right?

Reply:
Yes